instance/api/openedx_appserver.py

```diff
@@ -98,5 +98,5 @@ def make_active(self, request, pk):
             return Response(
                 {"error": "Cannot make an unhealthy app server active."}, status=status.HTTP_400_BAD_REQUEST
             )
-        app_server.instance.set_appserver_active(app_server.pk)
-        return Response({'status': 'App server updated.'})
+        set_appserver_active(app_server.pk)
+        return Response({'status': 'App server activation initiated.'})
```

Comment:
OK - I had not encountered these cases personally; pip doesn't catch failing installation scripts? It's hard to imagine the installation must have worked correctly if one of the scripts in the middle failed? Anyway, if the error is gone, we don't need to care about this now, but if we get something like that again, I'd like to understand why this happens and is still ok, before ignoring it. I would still be concerned about missing valid errors in the middle, if we start getting used to see tracebacks in the logs.

Thanks for the additional logging btw!